analysis/symbol-light-classes/src/org/jetbrains/kotlin/light/classes/symbol/methods/SymbolLightNoArgConstructor.kt

```diff
@@ -37,6 +43,19 @@ internal class SymbolLightNoArgConstructor(
         SymbolLightMemberModifierList(
             containingDeclaration = this,
             modifiersBox = InitializedModifiersBox(visibility),
+            annotationsBox =
+                if (functionSymbolPointer == null) {
+                    // default constructor
+                    EmptyAnnotationsBox
+                } else {
+                    // @JvmOverloads constructor
```

Comment:
I'd either dropped both comments or changed `@JvmOverloads constructor` into something else, as it is not about `@JvmOverloads`